spec/unit/recipes/default_spec.rb

```diff
@@ -5,18 +5,19 @@
 # Copyright (c) 2015 Irving Popovetsky, All Rights Reserved.
 
 require 'spec_helper'
+require 'mixlib/versioning' # new chef-ingredient makes heavy use of this library
```

Comment:
This is how we worked around a similar issue for `chef-sugar`:
https://github.com/chef-cookbooks/omnibus/blob/master/.travis.yml#L21

You could obviously also include a `Gemfile` in the repo but in the case above we wanted to avoid introducing a Gemfile that might cause confusion.